uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -87,7 +87,7 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
 
     // Let's slide out a root branch now
     intelliJ.machete.runJs("project.slideOutBranch('develop')")
```

Comment:
Also, while you're here... this suite is now very inconsistent wrt. whether `runJs` or a dedicated method of `GitMacheteExtension.MacheteExtensions.machete` is invoked... maybe pls migrate the remaining `runJs` invocations to dedicated methods, so that `runJs` itself can be made private (?)